docs/running-on-kubernetes.md

```diff
@@ -507,6 +507,13 @@ See the [configuration page](configuration.html) for information on Spark config
     configuration properties e.g. <code>spark.kubernetes.namespace</code>.
   </td>
 </tr>
+<tr>
+  <td><code>spark.kubernetes.driver.master</code></td>
+  <td><code>https://kubernetes.default.svc</code></td>
+  <td>
+    The internal master address to be used for driver to request executors.
```

Comment:
I suggest change `internal master address` to `internal Kubernetes master (API server) address`.